docs/deploy-with-ICP.md

```diff
@@ -21,6 +21,6 @@ $ cd Scalable-Cassandra-deployment-on-Kubernetes
 ```
 
 From here, you can follow along with the instructions in the
-[root](../#1-create-a-cassandra-headless-service) of this repo to deploy
+[root](../README.md#1-create-a-cassandra-headless-service) of this repo to deploy
```

Comment:
This is fine, but it may be better to point directly to README.md on master?

```markdown
[root](https://github.com/IBM/Scalable-Cassandra-deployment-on-Kubernetes/blob/master/README.md#1-create-a-cassandra-headless-service)
```